### ISSUE TITLE
RSS feed: Duplicate entries

### ISSUE THREAD

**Issue by sbaack** [MozillaFoundation/foundation.mozilla.org#10990]
### Describe the bug
The RSS feed for the blog starts showing duplicate entries after it is being updated.

### To Reproduce
1. Add https://foundation.mozilla.org/en/blog/rss/ to a feed reader of your choice.
1. When newly added, the feed won't directly include duplicates. Wait for new articles to be published and refresh. Duplicates will start to appear and accumulate over time.

### Expected behavior
The RSS feed should contain each article only once (when the related article is published).

### Screenshots
![](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/8274140/87d48ec8-27f2-4aad-beef-5de6f6ac5db4)

### Desktop (please complete the following information):
Tested in Inoreader and Thunderbird.

### Additional context
Not sure if this is related to #8067

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/TP1-183)

**Comment by mtdenton** [comment]
@sbaack I'm subbed on Feedly and I'm not seeing this. However, I've re-added and I'll check back later. Note that we also have https://foundation.mozilla.org/blog/atom/ which sends out the same content but it sounds like that's not the case here.

**Comment by sbaack** [comment]
@mtdenton Thanks for the response. I just checked, there are still duplicates in Inoreader and Thunderbird for me. I'll test the atom feed as well

**Comment by simont-cr** [comment]
@sbaack Can you please let us know if this is still an issue or can be closed? Thanks!

**Comment by sbaack** [comment]
Still an issue. The normal RSS contains duplicates. I checked the ATOM feed again and that one doesn't seem to contain duplicates, but often doesn't display images, which might be a separate issue.

**Comment by simont-cr** [comment]
Thank you for your confirmation, we will add this to our list of items to consider for this year maintenance work.

**Comment by data-sync-user** [comment]
➤ Simon Acosta Torres commented:

Tyler Denton Initially you mentioned that you had issues on replicating the problem and it has been some time since the latest update. If you were to check and still not able to see the issue, I say that we should close.
